core/templates/pages/contributor-dashboard-page/services/translate-text.service.ts

```diff
@@ -62,26 +66,26 @@ export class TranslateTextService {
   STARTING_INDEX = -1;
   PENDING = 'pending';
   SUBMITTED = 'submitted';
-  stateWiseContents = {};
-  stateWiseContentIds = {};
-  stateNamesList = [];
-  stateAndContent = [];
+  stateWiseContents: StateNamesToContentIdMapping = {};
+  stateWiseContentIds: Record<string, string[]> = {};
+  stateNamesList!: string[];
+  stateAndContent: StateAndContent[]=[];
   activeIndex = this.STARTING_INDEX;
-  activeExpId;
-  activeExpVersion;
-  activeContentId;
-  activeStateName: string;
-  activeContentText: string;
-  activeContentStatus: Status;
+  activeExpId!: string;
+  activeExpVersion!: string;
+  activeContentId!: string;
+  activeStateName!: string;
+  activeContentText?: string | string[];
+  activeContentStatus?: Status;
 
   constructor(
     private translateTextBackedApiService:
       TranslateTextBackendApiService
   ) { }
 
-  private _getNextText(): string | string[] {
+  private _getNextText(): string | string[] | undefined {
```

Comment:
`null` seems better here than undefined. If that's not the case please add a comment above function

Reply:
Done.